datalab/datalab_session/analysis/line_profile.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from skimage.measure import profile_line
+
+# For creating an array of brightness along a user drawn line
+def line_profile(input, sci_hdu):
```

Comment:
I noticed you had type hints in some places but not others, can we add type hints to these functions? Just helps future you and future me